### PR TITLE
updated to 0.8

### PR DESCRIPTION
**Pull request by morethanreal**

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -12,16 +11,22 @@
     "seed"
   ],
   "main": "seed-element.html",
+  "repository": {
+    type: "git",
```
Comment (ebidel) [inline bower.json]: i think bower.json gives problems for properties without quotes

**Comment by ebidel** [comment]
@nevir started a branch as well: https://github.com/PolymerLabs/seed-element/tree/0.8-preview

**Comment by morethanreal** [comment]
@ebidel Yes, I want to merge this branch into his branch instead of pushing commits directly to it.

**Comment by nevir** [comment]
Alrighty, I've merged my branch, and I think I have mixed in all your changes (or their intent) via f4e9b7373bd8abd1039806a378726a442512a811 - mind reviewing that commit?

**Comment by nevir** [comment]
Closing this for now (since I think it's covered by above)

**Pull request closed without merging**